### PR TITLE
add gitlab ci sample config

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

**Comment by juanpicado** [comment]
not sure, but I think this should be here https://github.com/pnpm/pnpm.github.io/blob/main/docs/continuous-integration.md doesn't ? @zkochan ?

**Comment by zkochan** [comment]
it should be in both places

File: versioned_docs/version-6.x/continuous-integration.md [hunk 1]
```diff
@@ -119,3 +119,26 @@ jobs:
...
+
+```yaml title=".gitlab-ci.yml"
+stages:
+  - build
+
+build:
+  stage: build
+  image: node:14.16.0-buster
+  before_script:
+    - npm i -g pnpm
```
Comment (zkochan) [inline versioned_docs/version-6.x/continuous-integration.md:136]: The recommended way would be
```suggestion
    - curl -f https://get.pnpm.io/v6.js | node - add --global pnpm@6
```

**Review by zkochan — CHANGES_REQUESTED** [review]

**Comment by anbraten** [comment]
I updated the installation command and added the documentation part to the "default" documentation.

**Review by zkochan — APPROVED** [review]

**Pull request merged**